### PR TITLE
CAROLYN IS LEARNING...

### PR DESCRIPTION
**Pull request by LynW**

### CODE REVIEW

File: public/core/css/core.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@import url(https://fonts.googleapis.com/css?family=Quicksand|Chewy|Russo+One);
+
+body {
+  font-size: 100px;
```
Comment (toiletpapar) [inline public/core/css/core.css]: Big font size :o

**Comment by toiletpapar** [comment]
Yay learning!

**Comment by LynW** [comment]
Mooo

**Comment by LynW** [comment]
Moo

File: public/core/css/core.css (hunk 1 above)
Comment (LynW) [inline public/core/css/core.css]: NOT BIG ENOUGH!

**Pull request merged**